### PR TITLE
Fixed typo

### PR DESCRIPTION
**Pull request by netmilk**

### CODE REVIEW

File: src/interactive-config.coffee [hunk 1]
```diff
@@ -72,7 +72,7 @@ interactiveConfig.prompt = (config = {}, callback) ->
           (answers['apiary'] == true && ! config['custom']?['apiaryApiName']?) &&
           (answers['apiary'] == true && answers['apiaryApiKey'] != '')
         )
-      else
+      elsebe
```
Comment (w-vi) [inline src/interactive-config.coffee]: Whats this doing? :)

Comment (netmilk) [inline src/interactive-config.coffee]: Lol

**Comment by honzajavorek** [comment]
@netmilk I think it is worth squashing these two commits, but YMMV

**Comment by kylef** [comment]
Huge :+1: on squashing these commits, it's a good practise to have since otherwise [git-bisect](https://git-scm.com/docs/git-bisect) is difficult to use.

**Comment by honzajavorek** [comment]
Squashed the commits for @netmilk, rebased with `master` & merging. :balloon:

**Pull request merged**